From 187f2f4fbb67c8b81d298ba88f83d1991035a49a Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Fri, 31 Aug 2012 12:34:46 +0800
Subject: [PATCH 053/304] ENGR00222016 wifi:re-place wifi's softmac file

Before added vendor partition used to storage vendor data like wifi MAC,
this name is defaultly used by google which symlink to system/vendor and be
used by APPs.So rename "vendor" to "device".

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 arm2_6dq/init.rc     | 2 +-
 arm2_6sl/init.rc     | 2 +-
 imx5x/init.rc        | 2 +-
 imx6/etc/init.rc     | 4 ++--
 imx6/sabresd/init.rc | 2 +-
 sabreauto_6q/init.rc | 2 +-
 6 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/arm2_6dq/init.rc b/arm2_6dq/init.rc
index 976275e..f06dd59 100755
--- a/arm2_6dq/init.rc
+++ b/arm2_6dq/init.rc
@@ -71,4 +71,4 @@ on fs
     mount ext4 /dev/block/mmcblk0p5 /system ro remount
     mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc
     mount ext4 /dev/block/mmcblk0p6 /cache nosuid nodev
-    mount ext4 /dev/block/mmcblk0p8 /vendor nosuid nodev
+    mount ext4 /dev/block/mmcblk0p8 /device ro nosuid nodev
diff --git a/arm2_6sl/init.rc b/arm2_6sl/init.rc
index 0bb5e5e..6fc6083 100755
--- a/arm2_6sl/init.rc
+++ b/arm2_6sl/init.rc
@@ -70,4 +70,4 @@ on fs
     mount ext4 /dev/block/mmcblk0p5 /system ro remount
     mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc
     mount ext4 /dev/block/mmcblk0p6 /cache nosuid nodev
-    mount ext4 /dev/block/mmcblk0p8 /vendor nosuid nodev
+    mount ext4 /dev/block/mmcblk0p8 /device ro nosuid nodev
diff --git a/imx5x/init.rc b/imx5x/init.rc
index 1182531..13b4284 100644
--- a/imx5x/init.rc
+++ b/imx5x/init.rc
@@ -42,7 +42,7 @@ loglevel 3
 
 # Right now vendor lives on the same filesystem as system,
 # but someday that may change.
-    mkdir /vendor 0771 system system
+    mkdir /device 0771 system system
 
 # Create cgroup mount point for cpu accounting
     mkdir /acct
diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 4af4404..17cbd4c 100644
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -42,7 +42,7 @@ loglevel 3
 
 # Right now vendor lives on the same filesystem as system,
 # but someday that may change.
-#    symlink /system/vendor /vendor
+    symlink /system/vendor /vendor
 
 # Create cgroup mount point for cpu accounting
     mkdir /acct
@@ -60,7 +60,7 @@ loglevel 3
     mkdir /cache 0770 system cache
     mkdir /config 0500 root root
 
-    mkdir /vendor 0771 system system
+    mkdir /device 0771 system system
 
     # Directory for putting things only root should see.
     mkdir /mnt/secure 0700 root root
diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index 99d709e..c59c800 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -132,4 +132,4 @@ on fs
     mount ext4 /dev/block/mmcblk0p5 /system ro remount
     mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc errors=panic
     mount ext4 /dev/block/mmcblk0p6 /cache nosuid nodev
-    mount ext4 /dev/block/mmcblk0p8 /vendor nosuid nodev
+    mount ext4 /dev/block/mmcblk0p8 /device ro nosuid nodev
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index 53a09a6..57eb2d1 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -67,4 +67,4 @@ on fs
     mount ext4 /dev/block/mmcblk0p5 /system ro remount
     mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc
     mount ext4 /dev/block/mmcblk0p6 /cache nosuid nodev
-    mount ext4 /dev/block/mmcblk0p8 /vendor nosuid nodev
+    mount ext4 /dev/block/mmcblk0p8 /device ro nosuid nodev
-- 
1.8.0

